packages/web-api/src/methods.ts

```diff
@@ -151,6 +151,20 @@ export interface AdminTeamsSettingsSetNameArguments extends WebAPICallOptions, T
   team_id: string;
   name: string;
 }
+export interface AdminUsergroupsAddChannelsArguments extends WebAPICallOptions, TokenOverridable {
+  usergroup_id: string;
+  team_id: string;
+  channel_ids: string | string[];
```

Comment:
This can be out-of-scope of this PR but `methods.ts` is not consistent regarding the type of "ids" argument (e.g., channel_ids, users). Some are `string` while others are `string[]`.

Regarding this endpoint, Slack API works with either `"C123,C234"` (the surely compatible way with the API docs) or `["C123", "C234"]`. Doing the same for other endpoints may work but I haven't checked yet. 

@stevengill @aoberoi 
To make this consistent, we can modify the logic in `WebClient#serializeApiCallOptions` to consider an array object (=this means do `channel_ids.join(",")` instead of `JSON.stringify` anyway) and change some argument types to `string | string[]`. I think it's totally fine. Any thoughts?

Reply:
sounds like a good change to me